GMG/Scenes/Home/HomeView.swift

```diff
@@ -168,55 +174,114 @@ struct HomeView: View {
+                            .font(Typography.WantedSansStd.R4)
+                            .foregroundStyle(Color.white1)
+                            .padding(.bottom, 4)
+                            .lineLimit(1)
+                    }
+                    
                     Text("\(score.key.description) Key")
                         .font(Typography.WantedSansStd.R2)
                         .foregroundStyle(Color.white1)
-
+                    
                     Spacer()
-
+                    
                     Text(HomeView.dateConverter(score.createdAt))
                         .font(Typography.WantedSansStd.R2)
                         .foregroundStyle(Color.black6)
                 }
-
+                
                 Spacer()
-
+                
                 VStack(alignment: .trailing, spacing: .zero) {
-                    Button(
-                        action: {
-                            //TODO: 미트볼버튼 기능 추가
-                        },
-                        label: {
-                            Image(systemName: "ellipsis")
-                                .foregroundStyle(Color.white1)
+                    
+                    Menu {
+                        Button {
+                            startRename()
+                        } label: {
+                            HStack(spacing: 14) {
+                                Image(systemName: "pencil")
+                                    .foregroundStyle(Color.black1)
+                                Text("Rename")
+                                    .font(Typography.WantedSansStd.R3)
+                                    .foregroundStyle(Color.black1)
+                            }
                         }
-                    )
-                    .buttonStyle(.plain)
-
+                        
+                        Button {
+                            router.push(.export)
+                        } label: {
+                            HStack(spacing: 14) {
+                                Image(systemName: "square.and.arrow.up")
+                                    .foregroundStyle(Color.black1)
+                                Text("Export")
+                                    .font(Typography.WantedSansStd.R3)
+                                    .foregroundStyle(Color.black1)
+                            }
+                        }
+                        
+                        Button {
+                            modelContext.delete(score)
+                        } label: {
+                            HStack(spacing: 14) {
+                                Image(systemName: "trash")
+                                    .foregroundStyle(Color.blue2)
+                                Text("Delete")
+                                    .font(Typography.WantedSansStd.R3)
+                                    .foregroundStyle(Color.blue2)
+                            }
+                        }
```

Comment:
메뉴에서는 커스텀 폰트랑 색상 지정이 안되어서 그냥 버튼 사용하면 될 것 같습니다!

```suggestion
                        Button("Rename", systemImage: "pencil") {
                            startRename()
                        }
                        
                        Button("Export", systemImage: "square.and.arrow.up") {
                            router.push(.export)
                        }
                        
                        Button("Delete", systemImage: "trash") {
                            modelContext.delete(score)
                        }
```